### PR TITLE
chore: demonstrate exporting OpenTelemetry logs to Google Cloud

### PR DESCRIPTION
**Pull request by agrawal-siddharth**
Monitoring

### CODE REVIEW

**Review by yirutang — APPROVED** [review]
Please wait for Java lib owner's approval.

File: samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java
Comment (PhongChuong) [inline samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java:1]: The general pattern for samples is to have a main file and a integration test file. E.g., see:
[SampleStorage.java](https://github.com/googleapis/java-bigquerystorage/blob/5c143a0d0e165fcacc5cf00dee5e1a391726dc5c/samples/snippets/src/main/java/com/example/bigquerystorage/StorageSample.java) and [QuickstartSampleIT.java](https://github.com/googleapis/java-bigquerystorage/blob/5c143a0d0e165fcacc5cf00dee5e1a391726dc5c/samples/snippets/src/test/java/com/example/bigquerystorage/QuickstartSampleIT.java#L50).

File: samples/install-without-bom/pom.xml
Comment (PhongChuong) [inline samples/install-without-bom/pom.xml:25]: These can move directly into the dep as it's only referenced once.

**Review by PhongChuong — COMMENTED** [review]
Overall looks good. Just some minor suggestions.

File: samples/install-without-bom/pom.xml
Comment (agrawal-siddharth) [reply samples/install-without-bom/pom.xml]: done

File: samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java
Comment (agrawal-siddharth) [reply samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java:1]: Hi, I have added the files

samples/snippets/src/main/java/com/example/bigquerystorage/ExportOpenTelemetry.java
samples/snippets/src/test/java/com/example/bigquerystorage/ExportOpenTelemetryIT.java

**Pull request merged**
